react/index.d.ts

```diff
@@ -74,7 +74,7 @@ declare namespace React {
     type ClassicFactory<P> = CFactory<P, ClassicComponent<P, ComponentState>>;
 
     interface DOMFactory<P extends DOMAttributes<T>, T extends Element> {
-        (props?: P & ClassAttributes<T>, ...children: ReactNode[]): DOMElement<P, T>;
+        (props?: P & ClassAttributes<T> | null, ...children: Array<ReactNode | undefined | null>): DOMElement<P, T>;
```

Comment:
`children` should remain `ReactNode[]` and `ReactNode` should be fixed to allow `undefined` and `null`.